### PR TITLE
refactor: deno lib: remove namespace, remove needless exports, add some jsdoc

### PR DESCRIPTION
**Pull request by hugocaillard**
- Improve files structure (remove namespace usage following deno lint recommendation https://lint.deno.land/?q=namespace#no-namespace)
- Don't export utility functions `red` and `green`
- Add some JSDoc to improve deno doc https://deno.land/x/clarinet@v1.0.6/index.ts

--- 

- Refactor / Improve `types.buff()` usage, fix: #786 
  - `types.buff(val: ArrayBuffer | string)` has been deprecated in favor of `types.buff(val: Uint8Array | string)`

Here's what the deprecation look like in VSCode
Screenshot 2023-01-25 at 16 28 53

### CODE REVIEW

File: components/clarinet-deno/index.ts [hunk 1]
```diff
@@ -166,28 +165,35 @@ export class Chain {
...
   }
 
+  /**
+   * Call a read-only function
+   * @param contract Address of the contract implementing the function
+   * @param method The read-only function to call
+   * @param args Arguments to pass as clarity values
+   * @param sender Address of the caller
+   * @returns The result of th
+   */
```
Comment (hugocaillard) [inline components/clarinet-deno/index.ts:175]: @mefrem 
Added JSDoc for this function.
It'll appear in the generated doc and could be added to more functions (`getAssetsMaps`, `callContract` etc)

**Review by lgalabru — APPROVED** [review]
LGTM, thanks @hugocaillard!

**Comment by hugocaillard** [comment]
I simplified this PR in order to make it ready for review.
It originally included a `types.stringToBuff` methods, but will need to revisit it in the future.
This PR still includes needed changes (JS Docs, files structure, etc)

**Review by lgalabru — APPROVED** [review]
Thanks @hugocaillard!

**Pull request merged**